iam_policies.tf

```diff
@@ -386,6 +386,20 @@ data "aws_iam_policy_document" "shepherd_engineers" {
     }
   }
 
+  // Allow acm:* for certificate creation and destruction 
+  statement {
+    effect = "Allow"
+    actions = [
+      "acm:*",
```

Comment:
Instead of adding a whole new statement just add this to the list of `*` actions below in the other statement.